recipes/menuinst/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "1.4.8" %}
```

Comment:
It looks like `defaults` provides `1.4.10`. However only `1.4.8` is marked as latest. Raised issue ( https://github.com/ContinuumIO/menuinst/issues/57 ) for clarification.

Reply:
It does look like the tags for 1.4.9 and 1.4.10 are listed. I think "latest" in this case is in the github release sense of the word...so I'd bump it up to `1.4.10`

Reply:
Expect you are probably right here. Am probably being overly thorough/cautious.